src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -22,16 +25,37 @@ export default function WebexRemoteMedia({className, meetingID}) {
     remoteShare,
     error,
   } = useMeeting(meetingID);
+  const members = useMemberships(meetingID, 'meeting');
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
   const shareRef = useStream(remoteShare);
+  const hasOtherMembers = members.length > 1;
   const hasMedia = !!(remoteAudio || remoteVideo || remoteShare);
   const hasTwoMedia = remoteVideo && remoteShare;
   const mainClasses = {
     [`${WEBEX_COMPONENTS_CLASS_PREFIX}-remote-media`]: true,
     [className]: !!className,
     'split-screen': hasTwoMedia,
   };
+  let remoteDisplay;
+
+  if (hasMedia) {
```

Comment:
Is the `if` necessary? I doesn't affect what the JSX in `remoteDisplay` will be right?
Additionally, you have the check [in line 68](https://github.com/webex/components/pull/121/files#diff-e17d52c72e28c31e65dc68aa7ddf2314R68)

Reply:
Is it necessary? No. It saves a couple of memory allocations. It could make it easier for the next person looking at this to see `remoteDisplay` will not be rendered if there is no media.

Reply:
I actually thought it was confusing because the real check to render or not is in line 68, so it felt out of place